### PR TITLE
Rename input sld to input style

### PR DESCRIPTION
**Pull request by index-git**
- [x] Tests
- [x] Layman Test Client
- [x] Changlelog
- [x] Documentation

### CODE REVIEW

File: src/layman/upgrade/upgrade_v1_10_test.py [hunk 1]
```diff
@@ -171,3 +180,28 @@ def test_migrate_maps_on_wms_workspace(ensure_map):
...
+        input_style_dir = os.path.join(layer_fs_util.get_layer_dir(workspace, layer),
+                                       'input_style')
+
+        assert input_style_dir == input_style.get_layer_input_style_dir(workspace, layer)
+
+        ensure_layer(workspace, layer)
+
+        assert os.path.exists(input_sld_dir)
+        assert not os.path.exists(input_style_dir)
+
```
Comment (index-git) [inline src/layman/upgrade/upgrade_v1_10_test.py:201]: Assert also style file existence.

File: src/layman/upgrade/upgrade_v1_10_test.py [hunk 2]
```diff
@@ -171,3 +180,28 @@ def test_migrate_maps_on_wms_workspace(ensure_map):
...
+        upgrade_v1_10.migrate_input_sld_directory_to_input_style()
+
+        assert not os.path.exists(input_sld_dir)
+        assert os.path.exists(input_style_dir)
+
```
Comment (index-git) [inline src/layman/upgrade/upgrade_v1_10_test.py:206]: Assert also style file existence.

File: src/layman/layer/style.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
+from collections import namedtuple
+import xml.etree.ElementTree as ET
+
+from layman import LaymanError
+
+
+StyleTypeDef = namedtuple('StyleTypeDef', ['code',
+                                           'rootElement',
+                                           'extension',
+                                           'source',
```
Comment (index-git) [inline src/layman/layer/style.py:10]: Remove 'source' for now.

File: src/layman/layer/style.py [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                                'qgis',
+                                'layman.layer.qgis.style'
+                                ),
+                   ]
+
+
+def get_style_type_from_xml_file(style_path):
+    xml_tree = ET.parse(style_path)
+    root = xml_tree.getroot()
+    _, _, root_attribute = root.tag.rpartition('}')
```
Comment (index-git) [inline src/layman/layer/style.py:28]: Find a way to get an element name without string operation.

File: src/layman/layer/style.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+from layman import LaymanError
+
+
+StyleTypeDef = namedtuple('StyleTypeDef', ['code',
+                                           'rootElement',
+                                           'extension',
+                                           'source',
+                                           ])
+STYLE_TYPES_DEF = [StyleTypeDef('sld',
```
Comment (index-git) [inline src/layman/layer/style.py:12]: Move into `layman.layer.__init__.py`

File: src/layman/layer/style.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                                ),
+                   StyleTypeDef('qgis',
+                                'qgis',
+                                'qgis',
+                                'layman.layer.qgis.style'
+                                ),
+                   ]
+
+
+def get_style_type_from_xml_file(style_path):
```
Comment (index-git) [inline src/layman/layer/style.py:25]: Move to `layman.layer.filesystem.input_style.py`

**Review by jirik — CHANGES_REQUESTED** [review]

**Review by jirik — APPROVED** [review]

**Pull request merged**